### PR TITLE
🚀🚀🚀 Release 0.36.0

### PR DESCRIPTION
**Pull request by t-eckert**
Let's release 0.36.0

Changes proposed in this PR:
- Update `imageK8S` in `values.yaml` to `0.36.0`
- Update `image` in `values.yaml` to `1.11.0`

### CODE REVIEW

File: charts/consul/values.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: "hashicorp/consul:1.10.3"
+  image: "hashicorp/consul:1.11.0"
```
Comment (ishustava) [inline charts/consul/values.yaml:88]: this should be the latest 1.10.x since there's no 1.11 GA yet

File: charts/consul/values.yaml [hunk 2]
```diff
@@ -105,7 +105,7 @@ global:
   # image that is used for functionality such as catalog sync.
   # This can be overridden per component.
   # @default: hashicorp/consul-k8s-control-plane:<latest version>
-  imageK8S: "hashicorp/consul-k8s-control-plane:0.35.0"
+  imageK8S: "hashicorp/consul-k8s-control-plane:0.35.1"
```
Comment (ishustava) [inline charts/consul/values.yaml]: This should be 0.36. I think there are too many improvements in the changelog for a patch release.

**Review by ishustava — COMMENTED** [review]
We also need to update chart.yaml

File: charts/consul/values.yaml (hunk 1 above)
Comment (thisisnotashwin) [inline charts/consul/values.yaml:88]: This should still be consul 1.10.3
Consul 1.11.0 will only be available as a beta and users will have to explicitly update the values file to use the beta2 image.

**Comment by t-eckert** [comment]
Closing this as I mistakenly thought the next version would be `0.35.1`. Version is actually `0.36.0` and I will open up a new branch/PR for that.

File: charts/consul/values.yaml (hunk 2 above)
Comment (t-eckert) [reply charts/consul/values.yaml]: Cool, this was a bad assumption by me. I had mapped `0.35.0` to beta 1 so I figured `0.35.1` would be beta 1.

**Pull request closed without merging**